src/azure-cli/azure/cli/command_modules/acr/__init__.py

```diff
@@ -13,7 +13,8 @@ class ACRCommandsLoader(AzCommandsLoader):
     def __init__(self, cli_ctx=None):
         from azure.cli.core.profiles import ResourceType
         super(ACRCommandsLoader, self).__init__(cli_ctx=cli_ctx,
-                                                resource_type=ResourceType.MGMT_CONTAINERREGISTRY)
+                                                resource_type=ResourceType.MGMT_CONTAINERREGISTRY,
+                                                operation_group='webhooks')
```

Comment:
We just want `2020-11-01-preview` as default, the webhooks is belong to `2020-11-01-preview`.
You can refer to other modules that use multiple versions. like monitor

Reply:
oh so by choosing webhooks, because it isn't in one of the other profiles like agent_pools? If so can we use a more general operation_group name like `registries`?

```
            'agent_pools': '2019-06-01-preview',
            'tasks': '2019-06-01-preview',
            'task_runs': '2019-06-01-preview',
            'runs': '2019-06-01-preview',
```

Reply:
```
        ResourceType.MGMT_CONTAINERREGISTRY: SDKProfile('2020-11-01-preview', {
            'agent_pools': '2019-06-01-preview',
            'tasks': '2019-06-01-preview',
            'task_runs': '2019-06-01-preview',
            'runs': '2019-06-01-preview',
        }),
```
In this profile, we defined a default apiversion and some other groups using special version.
But in the base class ACRCommandsLoader, it doesn't know which one should be used as default.
Why we use `webhooks`? Actually we use other group is ok. Because it will use default apiversion `2020-11-01-preview` when group name is not in special group list defined in profile.
I just want to tell ACRCommandsLoader to use `2020-11-01-preview` as default version.

Reply:
What happens if we don't set `operation_group='webhooks'`? What is the default version in this case?

Reply:
When we run `get_models`, we need a parameter `operation_group` to tell `get_models` to use which model in which api version, like this [line](https://github.com/Azure/azure-cli/blob/dev/src/azure-cli/azure/cli/command_modules/vm/custom.py#L1760).
If we don't clarify using which api_version/operation_group, then it will find it from [AzLoader](https://github.com/Azure/azure-cli/blob/dev/src/azure-cli/azure/cli/command_modules/vm/__init__.py#L29)

This is why we need this `operation_group` here. 

In monitor, it rarely uses `get_models` and always clarify `operation_group` when using. This why you don't find `operation_group` in AzLoader of monitor.

Reply:
So when `operation_group='webhooks'`  is not set, `get_models` can't get the default api-version of `ResourceType.MGMT_CONTAINERREGISTRY`, right?